python/tskit/trees.py

```diff
@@ -3587,52 +3587,62 @@ def breakpoints(self, as_array=False):
             breakpoints = map(float, breakpoints)
         return breakpoints
 
-    def at(self, position):
+    def at(self, position, **kwargs):
         """
         Returns the tree covering the specified genomic location. The returned tree
         will have ``tree.interval.left`` <= ``position`` < ``tree.interval.right``.
         See also :meth:`Tree.seek`.
 
+        :param float position: A genomic location.
+        :param \\**kwargs: Further arguments used as parameters when constructing the
+            returned :class:`Tree`
```

Comment:
I didn't do this in case of future changes to the actual kwargs list for creating a `Tree` (it's changed in the past, and it might be a faff changing it in multiple places). But I guess that's fine, so I'll just add that phrase into all 4 docstings, @benjeffery

Reply:
Yeah, I debated it too as we will need to maintain the docstring, but I think it's worth it.